evaluation.py

```diff
@@ -110,10 +120,10 @@ def idxs_to_sentences(predictions, vocab, src_sents = None, copy_lut = None, att
                 break
             elif word in (BOS_TOKEN, PAD_TOKEN):
                 continue
-            elif word is UNK_TOKEN and type(src_sents) != type(None) and type(attn) != type(None) and type(copy_lut) != type(None):
+            elif word is UNK_TOKEN and type(src_sents) != type(None) and\
+                type(attn) != type(None) and type(copy_lut) != type(None):
```

Comment:
My opinion on formatting (no changes needed): I prefer the first
```
if (
    expr_one
    and expr_two
):
if expr_one\
    and expr_two:
```

Reply:
Sure I can do that!